contrib/bindings/go/pathrs/capi.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) 2019, 2020 Aleksa Sarai <cyphar@cyphar.com>
+// Copyright (C) 2019, 2020 SUSE LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package pathrs
+
+/*
+#cgo LDFLAGS:-L ../../../../target/debug/ -l pathrs
+#include "../../../../include/pathrs.h"
```

Comment:
This won't work outside of the `libpathrs` repo. To be fair, we need to work around this somehow but the Python bindings do handle `LD_LIBRARY_PATH` properly. I would suggest we should fix the pkg-config issue (#13) first so that we can just do pkg-config magic for the source tree and make these bindings work for system installs of `libpathrs`.

Reply:
All right it wont and that is what I truly dislike here, but from my perspective it cannot be handled easily? (I have not found anything yet to fix it)

This comment is associated with https://github.com/openSUSE/libpathrs/pull/22#discussion_r368209742

Reply:
Apparently we can remove it and instead force user to set env variables
```bash
export CGO_LDFLAGS "-L/path/to/shared/lib -lpathrs"
export CGO_CFLAGS "-I/path/to/include"
```

Reply:
Yeah, you can use the `CGO_*` flags -- but in the long term I would prefer we had:

```
// #cgo pkg-config: pathrs
// #include <pathrs.h>
```

For now, how about you just have:

```
// #cgo LDFLAGS: -lpathrs
// #include <pathrs.h>
```

And the other LDFLAGS can be set in the environment. You also will need to set `CGO_CFLAGS=-I../../../../include/` or similar.

Reply:
Yeah, you can use the `CGO_*` flags -- but in the long term I would prefer we had:

```
// #cgo pkg-config: pathrs
// #include <pathrs.h>
```

For now, how about you just have:

```
// #cgo LDFLAGS: -lpathrs
// #include <pathrs.h>
```

And the other LDFLAGS can be set in the environment. You also will need to set `CGO_CFLAGS=-I../../../../include/` or similar.